packages/react-component-manifests/src/manifests/Slider/Slider.tsx

```diff
@@ -235,29 +170,48 @@ const Slider = forwardRef<
             top: "50%",
             transform: "translate(0px, -50%)",
             // center of thumb should match the starting point of track
-            width: thumbPosition,
-            left: radius,
+            width: `calc(${((value - minValue) / valueRange) * 100}% - 2 * ${radius})`,
+            left: 0,
           }}
           onClick={onTrackClicked}
         ></div>
         {/** thumb */}
         <div
+          className="slide-thumb outline"
           ref={thumbRef}
           style={{
             position: "absolute",
-            left: thumbPosition,
+            left: `calc(${((value - minValue) / valueRange) * 100}% - 2 * ${radius})`,
             height: `calc(2 * ${radius})`,
             width: `calc(2 * ${radius})`,
-            backgroundColor: thumbColor,
+            backgroundColor:((thumbVariant === ThumbVariant.OUTLINED) ? '#fff' : thumbColor),
             top: "50%",
             transform: `translate(0px, -50%)`,
             borderRadius: "50%",
+            // Outline :
+            border: `3px solid ${thumbColor}`,
           }}
           onMouseDown={onMouseDown}
-        ></div>
+        >
+          <span className="thumb-value">{Math.round(value)}</span>
+        </div>
       </div>
     </>
   );
 });
 
 export default Slider;
+
+
+// Slider.defaultProps = {
+//   custom: {
+//     value: 50,
+//     maxValue: 100,
+//     minValue: 0,
+//     thickness: "4px",
+//     radius: "8px",
+//     trackColor: "#CCC",
+//     thumbColor: "#91d5ff",
+//     selectColor: "#91d5ff",
+//   },
+// };
```

Comment:
Please don't leave commented code around in a PR unless there is a strong reason.